src/main/java/com/reckue/post/models/repositories/VideoNodeRepository.java

```diff
@@ -0,0 +1,9 @@
+package com.reckue.post.models.repositories;
+
+import com.reckue.post.models.VideoNode;
+import org.springframework.data.mongodb.repository.MongoRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface VideoNodeRepository extends MongoRepository<VideoNode,String> {
```

Comment:
Add space after comma before String in generics.

Reply:
fixed)